pylintrc

```diff
@@ -34,6 +34,9 @@ load-plugins=
 # multiple time (only on the command line, not in the configuration file where
 # it should appear only once).
 disable=
+    missing-docstring,
```

Comment:
I agree about the downside of holding new code to the higher standard, but disagree about the utility of displaying "old" violations.

In this case, there are more than 2 dozen such violations. So as a developer, if I get a failing quality build, I now have to wade through twenty-some violations and try to figure which ones are worth paying attention to. I know this has previously been a pain point in the platform.

Subsequently, I just went ahead and added the missing docstrings, so it's a moot point now.